src/video/windows/SDL_windowswindow.c

```diff
@@ -573,8 +573,39 @@ WIN_HideWindow(_THIS, SDL_Window * window)
 void
 WIN_RaiseWindow(_THIS, SDL_Window * window)
 {
+    /* If desired, raise the window more forcefully.
+     * Technique taken from http://stackoverflow.com/questions/916259/ .
+     * Specifically, http://stackoverflow.com/a/34414846 .
+     *
+     * The issue is that Microsoft has gone through a lot of trouble to make it
+     * nearly impossible to programmatically move a window to the foreground,
+     * for "security" reasons. Apparently, the following song-and-dance gets
+     * around their objections. */
+    const char *hint = SDL_GetHint(SDL_HINT_FORCE_RAISEWINDOW);
```

Comment:
You probably want SDL_GetHintBoolean() here.

Reply:
Sorry, didn't know SDL_GetHintBoolean() existed. (I'm new to SDL.) Change made.